python/rmm/rmm/_lib/memory_resource.pyx

```diff
@@ -1188,3 +1189,10 @@ def get_log_filenames():
         else None
         for i, each_mr in _per_device_mrs.items()
     }
+
+
+def get_free_device_memory():
```

Comment:
Should we rename this function to match the C++ function `available_device_memory`? It can return a tuple of (free, total) like the C++ function.

Reply:
Done